### PR TITLE
Issue #145 - completed docstrings for cfg_accurate & veritesting

### PR DESCRIPTION
**Pull request by f-prettyland**
Continued work on docstring work, Issue #145.

I also broke up veristesting's execute_and_merge to break out merge to make it more readable, I hope you agree.

### CODE REVIEW

File: angr/analyses/veritesting.py [hunk 1]
```diff
@@ -333,76 +343,7 @@ def _execute_and_merge(self, path):
...
-                            loop_ctr >= self._loop_unrolling_limit + 1 for loop_ctr in
-                            path_group.one_merge_tmp.info['loop_ctrs'].itervalues()
-                        ):
-                            l.debug("... merged path is overlooping")
-                            path_group.move('merge_tmp', 'deadended')
-                        else:
-                            l.debug('... merged path going to active stash')
-                            path_group.move('merge_tmp', 'active')
-
+                path_goup = _join_merge_points(path_group, merge_points)
```
Comment (ltfish) [inline angr/analyses/veritesting.py]: `path_goup` -> `path_group`

And I think you missed a `self.` before `_join_merge_points(`.

**Comment by zardus** [comment]
CI failed for this, but I suspect it's a one of those intermittent failures. I'm retrying the tests to see if that's the case.

**Comment by zardus** [comment]
It looks like those failures aren't intermittent :-(. Gotta get some time to look into them...

**Comment by zardus** [comment]
I must have been looking at the wrong test log... The issues are just linting; I'm fixing them and rebasing this PR on master, then we'll get it merged in.

**Comment by f-prettyland** [comment]
Oh dang! Thanks for checking and fixing that

**Comment by zardus** [comment]
No problem. Thanks for the contribution!

**Pull request merged**